opal/mca/threads/argobots/threads_argobots.h

```diff
@@ -30,7 +30,7 @@
 
 static inline void opal_threads_argobots_ensure_init(void)
 {
-    if (ABT_initialized() != 0) {
+    if (ABT_initialized() != ABT_SUCCESS) {
```

Comment:
would you mind reversing the check to have ABT_SUCCESS != ABT_initialize()?  I know it looks weird but that's the "openmpi" coding style.

Reply:
Thank you. I will follow the Open MPI style. I will examine other checks.

Reply:
I found some did not follow.
```
if (NULL == t->t_run || t->t_handle 1= (pthread_t)-1) {
to
if (NULL == t->t_run || (pthread_t)-1 != t->t_handle) {
```
I fixed them as far as I could find.